config/atom/atom.go

```diff
@@ -52,9 +52,9 @@ func Open(name string) (File, error) {
 }
 
 type file struct {
-	to    string
-	tmp   *os.File
-	dirty bool
+	to       string
+	tmp      *os.File
+	writeErr bool
```

Comment:
I actually like `dirty`  as it hints it's a boolean and that the state is not right. When I read `writeErr`, I expect an error on first sight, but the value turns out to be a boolean.

Reply:
Agreed `writeErr` isn't quite right either.

Do you think any of the following  seem better than dirty to you?

* `errored`
* `writeErrored`
* `writeErrOccured`
* `writeFailed`
* `noCommit`
* `dontRename`
* `badWrite`
* `invalidState`

If not, I'll be happy to leave it as is or add a clarifying comment over it in the struct.

Reply:
I like `badWrite`, but to be honest, I don't think `dirty` is so confusing here. Maybe you can put the comment over it.